lib/disability_compensation/responses/claims_service_response.rb

```diff
@@ -18,6 +18,7 @@ class Claim
       attribute :base_end_product_code, String
       attribute :development_letter_sent, Boolean
       attribute :status, String
+      attribute :claim_date, String
```

Comment:
For some reason, this field was not being used when receiving the LH Benefits Claims response. It's a pretty useful field.

cc @sethdarragile6

Reply:
I'm not sure, either. I think that generally we didn't need it because the LH submit request doesn't accept a `claimDate`. I think the backup submission process needs it for backdated claims, but that process likely doesn't use the same BC service